src/libs/SidebarUtils.ts

```diff
@@ -331,7 +333,11 @@ function getOptionData(
               }
             : null;
     }
-    const lastActorDisplayName = hasMultipleParticipants && lastActorDetails?.accountID && Number(lastActorDetails.accountID) !== currentUserAccountID ? lastActorDetails.displayName : '';
+
+    const lastActorDisplayName =
```

Comment:
Having these inline makes it more and more hard to follow - have you considered abstracting these checks away to a few functions? You can even colocate them nearby if they're not really reusable, but having eg. `isCurrentActorLast()` etc will be less cognitively demanding. Not a blocker though, this lgtm :)